go/vt/sqlparser/ast_format.go

```diff
@@ -665,6 +665,27 @@ func (ct *ColumnType) Format(buf *TrackedBuffer) {
 		if ct.Options.Comment != nil {
 			buf.astPrintf(ct, " %s %v", keywordStrings[COMMENT_KEYWORD], ct.Options.Comment)
 		}
+		if ct.Options.Invisible != nil {
+			if *ct.Options.Invisible {
+				buf.astPrintf(ct, " %s", keywordStrings[INVISIBLE])
+			} else {
+				buf.astPrintf(ct, " %s", keywordStrings[VISIBLE])
+			}
+		}
+		switch ct.Options.Format {
+		case FixedFormat:
+			buf.astPrintf(ct, " %s %s", keywordStrings[COLUMN_FORMAT], keywordStrings[FIXED])
+		case DynamicFormat:
+			buf.astPrintf(ct, " %s %s", keywordStrings[COLUMN_FORMAT], keywordStrings[DYNAMIC])
+		case DefaultFormat:
+			buf.astPrintf(ct, " %s %s", keywordStrings[COLUMN_FORMAT], keywordStrings[DEFAULT])
```

Comment:
We could create a ToString function in ast_funcs file and use that here. The benefit is that if we use this column_format field in some other place, we won't be repeating this logic

Reply:
@GuptaManan100 Done!